src/coreclr/src/jit/compiler.h

```diff
@@ -7411,13 +7411,17 @@ class Compiler
     // Returns the frame size at which we will generate a loop to probe the stack.
     target_size_t getVeryLargeFrameSize()
     {
+        const target_size_t pageSize = eeGetPageSize();
 #ifdef TARGET_ARM
         // The looping probe code is 40 bytes, whereas the straight-line probing for
         // the (0x2000..0x3000) case is 44, so use looping for anything 0x2000 bytes
         // or greater, to generate smaller code.
-        return 2 * eeGetPageSize();
+        return 2 * pageSize;
+#elif defined(TARGET_ARM64)
+        constexpr target_size_t ldrLargestPositiveImmByteOffset = 0x8000;
```

Comment:
It would be worthwhile having a comment here (even a quite detailed comment) explaining this math, or at least a pointer to someplace else in the code that has such a comment.